test/fc_lapack/src/CMakeLists.txt

```diff
@@ -1,10 +1,15 @@
-if(LAPACK_FOUND)
-    if(CMAKE_Fortran_COMPILER_ID MATCHES PGI)
-        # we remove -rdynamic flag added by CMake
-        # PGI Fortran does not recognize this flag
-        SET(CMAKE_SHARED_LIBRARY_LINK_Fortran_FLAGS)
-    endif()
+if(CMAKE_Fortran_COMPILER_ID MATCHES PGI)
+    # we remove -rdynamic flag added by CMake
+    # PGI Fortran does not recognize this flag
+    SET(CMAKE_SHARED_LIBRARY_LINK_Fortran_FLAGS)
+endif()
 
+if(CMAKE_Fortran_COMPILER_ID MATCHES Intel AND ENABLE_STATIC_LINKING)
```

Comment:
Shall this remain in local src/CMakeListst.txt or should it go to compilers/COMPILER.Fortran.cmake  ?